include/plugin_api.inc.php

```diff
@@ -172,6 +173,7 @@ static function register_default_plugins()
         serendipity_plugin_api::create_plugin_instance('@serendipity_plugin_archives');
         serendipity_plugin_api::create_plugin_instance('@serendipity_plugin_categories');
         serendipity_plugin_api::create_plugin_instance('@serendipity_plugin_syndication');
+        serendipity_plugin_api::create_plugin_instance('@serendipity_plugin_subscribe');
```

Comment:
I'd be more comfortable with not making it a default plugin directly and having it one cycle as opt-in.